src/psyclone/tests/conftest.py

```diff
@@ -233,3 +234,22 @@ def fixture_fortran_reader():
 def fixture_fortran_writer():
     '''Create and return a FortranWriter object with default settings.'''
     return FortranWriter()
+
+
+@pytest.fixture(scope='function')
+def clear_module_manager_instance():
+    ''' The tests in this module all assume that there is no pre-existing
```

Comment:
docstring needs re-writing slightly now that it's in the top-level confteset.py.

Reply:
Done.